src/Metadata/Driver/AnnotationDriver.php

```diff
@@ -222,8 +222,6 @@ public function loadMetadataForClass(\ReflectionClass $class): ?BaseClassMetadat
                     } elseif ($annot instanceof XmlValue) {
                         $propertyMetadata->xmlValue = true;
                         $propertyMetadata->xmlElementCData = $annot->cdata;
-                    } elseif ($annot instanceof XmlElement) {
```

Comment:
why removing this?

Reply:
@goetas , maybe because it is defined a bit up, at [line#201](https://github.com/schmittjoh/serializer/pull/1232/files/630d6409f95c83acb8c9d9362f7e65b87ec774a5#diff-3666ddf8eb574f2b48494504f36390fe985a4c8b782cbc558d20cecc55949ed4R201)

Reply:
uu, right, thanks!